pkg/controller/grafana/grafana_reconcile.go

```diff
@@ -87,6 +87,13 @@ func reconcileGrafana(r *ReconcileGrafana, cr *v1alpha1.Grafana) error {
 		return err
 	}
 
+	err = cleanupCSMonitoring(r, cr)
+	if err != nil {
+		// no need to return error here as its just cleanup and no impact if it fails
+		log.Info("Fail to cleanup one or more old CS Monitoring resources.")
```

Comment:
please also emit kubernetes event and provide some ibm.biz short link to the docs providing the manual steps to do the cleanup

Reply:
Addressed this partially with kubernetes event. once we update the doc with manual cleanup instructions incase grafana-operator fails to clean, i will include the short link that has instructions.